app/views/profiles/index.html.erb

```diff
@@ -1,27 +1,49 @@
-<aside>
-  <% if @profile.nil? %>
-    <h3> <%= link_to current_user.name, new_profile_path %> </h3>
-  <% else %>
-    <h3> <%= link_to current_user.name, profile_path(current_user.id) %> </h3>
-    <% skill = Skill.find_by(user_id: current_user.id ) %>
-    <% if !skill.nil? %>
-      <h4> <%= skill.name %> </h4>
-    <% else %>
-      <h4> <%= link_to "Add skill", new_skill_path(current_user.id) %> </h4>
-    <% end %>
-  <% end %>
-</aside>
-<main><br>
-  <h2>Profiles</h2>
-  <% @profiles.each do |profile| %>
-    <% user = User.find_by(id: profile.user_id) %>
-    <% skill = Skill.find_by(user_id: user.id) %>
-    <h4> <%= link_to user.name, profile_path(user.id)%> </h4>
-    <% if !skill.nil? %>
-      <p> <%= skill.name %> </p>
-    <% end %>
-    <p> <%= profile.city %>, <%= profile.state %>, <%= profile.country %> </p>
-    <p> <%=  user.birthday.strftime("%m-%d-%Y") %> </p>
-    <p> <%= profile.description %> </p>
-  <% end %>
-</main>
+<div class="container">
+  <div class="feed">
+    <aside>
+      <div class="user">
+        <%= image_tag 'profile', class: 'user__img' %>
+        <% if @profile.nil? %>
+          <h3 class="user__name" > <%= link_to current_user.name, new_profile_path %> </h3>
+        <% else %>
+          <h3 class="user__name"> <%= link_to current_user.name, profile_path(current_user.id) %> </h3>
+          <% skill = Skill.find_by(user_id: current_user.id ) %>
+          <% if !skill.nil? %>
+            <h4 class="user__skill" > <%= skill.name %> </h4>
+          <% else %>
+            <h4 class="user__skill"> <%= link_to "Add skill", new_skill_path(current_user.id) %> </h4>
+          <% end %>
+        <% end %>
+      </div>
+    </aside>
```

Comment:
What do you think about moving this code to an external file?

Reply:
I agree with @YilKanda

Reply:
x3